### ISSUE TITLE
Not able to access https://packagecontrol.io nor Package Control functionality

### ISSUE THREAD

**Issue by CrisVM** [wbond/package_control#1511]
Not sure what's going on here.

Package control doesn't work at all on my computer.

When I try to use it, I always get this error message:
![ZION³ 2020-10-21 at 19 11 22](https://user-images.githubusercontent.com/3301474/96792687-46b08c00-13d1-11eb-86b5-3f8481a1cf42.png)

...and I'm not even able to open https://packagecontrol.io in a regular browser window:
![ZION³ 2020-10-21 at 19 12 26](https://user-images.githubusercontent.com/3301474/96792774-7790c100-13d1-11eb-852d-c56160ea4889.png)

...but it works perfectly fine if I access it through **Tor**:
![ZION³ 2020-10-21 at 19 14 29](https://user-images.githubusercontent.com/3301474/96792883-ac9d1380-13d1-11eb-984a-58741c3913a8.png)

I just enabled debug and invoked **Package Control: Install Package** again to generate the following log:

```
reloading settings Packages/User/Package Control.sublime-settings
Package Control: Settings change detected, clearing cache
Package Control: Fetching list of available packages and dependencies
  Platform: osx-x64
  Sublime Text Version: 3211
  Package Control Version: 3.3.0
Package Control: Download Debug
  URL: https://packagecontrol.io/channel_v3.json
  Timeout: 30
  Resolved IP: 50.116.34.243
  Resolved IPv6: 2600:3c02::f03c:91ff:fedf:6883
Package Control: Found previously exported CA bundle at /Users/crisvm/Library/Application Support/Sublime Text 3/Packages/User/oscrypto-ca-bundle.crt (285226 bytes)
Package Control: Curl Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Trying to execute command /usr/bin/curl --connect-timeout 30 -sSL --tlsv1 --dump-header /var/folders/w8/f0vrqdds6ws9k02tgb0kfmj80000gn/T/tmpth19zr --user-agent 'Package Control v3.3.0' --header 'Accept-Encoding: bzip2,gzip,deflate' --cacert '/Users/crisvm/Library/Application Support/Sublime Text 3/Packages/User/Package Control.merged-ca-bundle' -v https://packagecontrol.io/channel_v3.json
Package Control: Curl HTTP Debug General
    Trying 2600:3c02::f03c:91ff:fedf:6883...
  TCP_NODELAY set
    Trying 50.116.34.243...
  TCP_NODELAY set
  After 14998ms connect time, move on!
  connect to 2600:3c02::f03c:91ff:fedf:6883 port 443 failed: Operation timed out
  After 7398ms connect time, move on!
  connect to 50.116.34.243 port 443 failed: Operation timed out
  Failed to connect to packagecontrol.io port 443: Operation timed out
  Closing connection 0
Package Control: Error downloading channel. curl: (7) Failed to connect to packagecontrol.io port 443: Operation timed out downloading https://packagecontrol.io/channel_v3.json.
```

I live in São Paulo, Brazil. _(I think maybe this detail is relevant, 'cause I can access https://packagecontrol.io through Tor but not from a regular browser window)_

Any help will be really appreciated. Please let me know if you need further details.

Thanks in advance.
Cristiano.

**Comment by musterknabe** [comment]
I get them same error "There are no packages available..." since 6 weeks.

Thanks to your error log I could solve my problem.
Somehow in "Preferences>Settings>Package Settings>Package Control" my user settings had an empty "channels" array, preventing to get packages. I don't know how it got there, I never opened that settings file before.

I'm sorry this won't help you @CrisVM. Try it from a different network to make sure it's not your ISP blocking the domain packagecontrol.io

To the devs: Maybe check if the channels array is empty and give a better error message?

**Comment by musterknabe** [comment]
To recap, I probably called "Package Control: Remove Channel" via CMD + SHIFT + P by mistake.
Still an error message saying "You don't have any channels configured, do this..." would go a long way.

**Comment by JDBuscemi** [comment]
I'm getting the same error, except I can't even get packagecontrol.io to load at all... direct, vpn, tor, different machine... it's all a fail.  And I've got nothing untoward in my Sublime Text installation.  This is a fresh install from scratch on a new MacBook.

**Comment by CrisVM** [comment]
I've figured out what was going on.

Because of the remote work I do for a customer, I have to use a VPN client (Cisco Anyconnect).

Sometimes I just forget to disconnect from it. And that's why I couldn't even access the website from a regular browser window.

I'm closing this issue now.

Well, at least the log I've posted served to help @musterknabe to realise what happened to his computer as well... 😎

Cheers.
